tests/planner_test/tpch.planner.sql

```diff
@@ -112,12 +112,59 @@ order by
     l_linestatus;
 
 /*
-Projection: [l_returnflag, l_linestatus, sum(l_quantity), sum(l_extendedprice), sum((l_extendedprice * (1 - l_discount))), sum(((l_tax + 1) * (l_extendedprice * (1 - l_discount)))), (sum(l_quantity) / count(l_quantity)), (sum(l_extendedprice) / count(l_extendedprice)), (sum(l_discount) / count(l_discount)), rowcount] (cost=39238.984)
-  Order: [l_returnflag asc, l_linestatus asc] (cost=33158.984)
-    Aggregate: [sum(l_quantity), sum(l_extendedprice), sum((l_extendedprice * (1 - l_discount))), sum(((l_tax + 1) * (l_extendedprice * (1 - l_discount)))), count(l_quantity), count(l_extendedprice), sum(l_discount), count(l_discount), rowcount], groupby=[l_returnflag, l_linestatus] (cost=25300)
-      Projection: [l_quantity, l_extendedprice, l_discount, l_tax, l_returnflag, l_linestatus] (cost=18260)
-        Filter: (1998-09-21 >= l_shipdate) (cost=12900)
-          Scan: lineitem[l_quantity, l_extendedprice, l_discount, l_tax, l_returnflag, l_linestatus, l_shipdate] (cost=7000)
+
+Projection                                                                                                              
+├── cost: 39238.984                                                                                                     
+├── exprs:                                                                                                              
+│   ┌── l_returnflag                                                                                                    
+│   ├── l_linestatus                                                                                                    
+│   ├── sum {  }                                                                                                        
```

Comment:
The inner expression is missing. 😇

BTW, this empty brackets seems a little wide. 😇
How about `{}` or `{ }`?

Reply:
I would say it's probably a bug in the generator. I'll take a look